frontend/src/components/Board/BuyParts/index.tsx

```diff
@@ -408,6 +408,20 @@ function redirectToStore(retailer: Retailer) {
   form.submit()
 }
 
+/**
+  Escape a CSV value according to RFC 4180.
+  https://www.ietf.org/rfc/rfc4180.txt
+ */
+function escapeCsvValue(value: string | number | undefined) {
+  if (
+    typeof value === 'string' &&
+    (value.includes(',') || value.includes('"') || value.includes('\n'))
+  ) {
+    return `"${value.replace(/"/g, '""')}"`
```

Comment:
Is that right, shouldn't it be `\"`?

Reply:
I implemented it according to the RFC linked in the function docstring

![image](https://github.com/user-attachments/assets/ff85c919-33b8-48aa-a980-1524b3a70b75)